tests/contrib/django/asgi.py

```diff
@@ -1,4 +1,29 @@
+from channels.auth import AuthMiddlewareStack
+from channels.routing import ProtocolTypeRouter
+from channels.routing import URLRouter
 from django.core.asgi import get_asgi_application
+from django.urls import re_path
+
+from ddtrace.contrib.asgi import TraceMiddleware
 
 
 application = get_asgi_application()
+
+
+async def simple_asgi_app(scope, receive, send):
+    await send({"type": "http.response.start", "status": 200, "headers": [(b"Content-Type", b"text/plain")]})
+    await send({"type": "http.response.body", "body": b"Hello World. It's me simple asgi app"})
+
+
+channels_application = ProtocolTypeRouter(
+    {
+        "http": AuthMiddlewareStack(
+            URLRouter(
+                [
+                    re_path(r"simple-asgi-app/", TraceMiddleware(simple_asgi_app)),
```

Comment:
maybe we should have two routes here, one traced and one untraced, what do you think?

this is a good test that `TraceMiddleware` works as expected / doesn't have issues, but I wonder if we should have a route for just `simple_asgi_app` as well to verify nothing changed, nothing gets added/traced, etc.

what do you think?

Reply:
Yup. I just added that